core/server/api/v2/index.js

```diff
@@ -119,15 +103,39 @@ module.exports = {
         return shared.pipeline(require('./actions'), localUtils);
     },
 
-    get authorsPublic() {
-        return shared.pipeline(require('./authors-public'), localUtils, 'content');
-    },
-
     get site() {
         return shared.pipeline(require('./site'), localUtils);
     },
 
     get serializers() {
         return require('./utils/serializers');
+    },
+
+    /**
+     * Content API Controllers
+     *
+     * @NOTE:
+     *
+     * Please create separate controllers for Content & Admin API. The goal is to expose `api.v2.content` and
+     * `api.v2.admin` soon. Need to figure out how serializers & validation works then.
+     */
+    get pagesPublic() {
+        return shared.pipeline(require('./pages-public'), localUtils, 'content');
+    },
+
+    get tagsPublic() {
+        return shared.pipeline(require('./tags-public'), localUtils, 'content');
+    },
+
+    get publicSettings() {
```

Comment:
Needs renaming, not important now.

Reply:
Would be best tackled during https://github.com/TryGhost/Ghost/issues/10106, right?

Reply:
yes perf 👍